_includes/setup/get-sdk-win.md

```diff
@@ -0,0 +1,59 @@
+## Get the Flutter SDK
+
+To get Flutter, use `git` to clone the repository and then add the `flutter` tool to your path.
+Running `flutter doctor` shows any remaining dependencies you may need to install.
+
+### Clone the repo
+
+If this is the first time you're installing Flutter on this machine, clone the
+repository and then add the `flutter` tool to your path:
+
+```
+> git clone https://github.com/flutter/flutter.git
+> setx path "%path%;%cd%\flutter\bin"
+```
+
+To update an existing version of Flutter, see [Upgrading Flutter](/upgrading/)
+
+
+### Run flutter doctor
+
+Run the following command to see if there are any dependencies you need to install to complete
+the setup:
+
+```
+> flutter doctor
+```
+
+This command checks your environment and displays a report to the terminal window.
+The Dart SDK is bundled with Flutter; it is not necessary to install Dart separately.
+Check the output carefully for other software you may need to install or further 
+tasks to perform (shown in **bold** text).
```

Comment:
nit: The text will only be in bolt on more recent versions of Windows 10. Older versions of Windows 10 and old versions of Windows don't support bolt text in the terminal.